tests/chainerx_tests/unit_tests/routines_tests/test_activation.py

```diff
@@ -72,6 +73,44 @@ def forward_xp(self, inputs, xp):
 ]
 
 
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize(*(
+    # Special shapes
+    chainer.testing.product({
+        'shape': [(), (0,), (1,), (2, 0, 3), (1, 1, 1), (2, 3)],
+        'in_dtypes,out_dtype': _in_out_dtypes_math_functions,
+        'input': [-2, 2],
+        'contiguous': [None, 'C'],
+        'alpha_range': [(-2.0, 0.0), 0.0, (0.0, 2.0)],
+    })
+    # Special values
+    + chainer.testing.product({
+        'shape': [(2, 3)],
+        'in_dtypes,out_dtype': _in_out_float_dtypes_math_functions,
+        'input': [0, float('inf'), -float('inf'), float('nan')],
+        'skip_backward_test': [True],
+        'skip_double_backward_test': [True],
+        'alpha_range': [(-2.0, 0.0), 0.0, (0.0, 2.0)],
+    })
+))
+class TestElu(UnaryMathTestBase, op_utils.NumpyOpTest):
+
+    def setup(self):
+        if isinstance(self.alpha_range, tuple):
+            l, u = self.alpha_range
+            self.alpha = random.uniform(l, u)
+        else:
+            self.alpha = self.alpha_range
+
+    def func(self, xp, a):
+        if xp is numpy:
+            y = a.copy()
+            negzero_indices = y <= 0
+            y[negzero_indices] = self.alpha * numpy.expm1(y[negzero_indices])
+            return y
+        return xp.elu(a, self.alpha)
```

Comment:
Thank you for the fix.
Please also test "unspecified" alpha.
You could implement similarly to existing tests like [test_connection.py](https://github.com/chainer/chainer/blob/v7.0.0a1/tests/chainerx_tests/unit_tests/routines_tests/test_connection.py#L427) (define `Unspecified` class and parameterize it as `alpha_range`).

Reply:
Sure. Will modify the parametrizations accordingly.